tests/test_internal.py

```diff
@@ -25,24 +34,14 @@
                     cs.int_schema(ref='int'),
                 ],
             ),
-            cs.definitions_schema(
-                cs.list_schema(cs.definition_reference_schema('list_of_ints')),
-                definitions=[
-                    cs.list_schema(cs.definition_reference_schema('int'), ref='list_of_ints'),
-                    cs.int_schema(ref='int'),
-                ],
-            ),
```

Comment:
Removing the second item in each test since there's no longer two steps, just one.